slippi-stats.js

```diff
@@ -217,157 +261,196 @@ for (i = 0; i < files.length; i++) {
         // If the player didn't quit out AND has more kills than the opponent, the same but with a lower percent, or the opponent quits out: it's a win, otherwise it's a loss. Ties handled above
         // if (!end_player_LRAS && (end_more_kills || end_lower_percent || end_opponent_LRAS)) {
         if (end_more_kills || end_lower_percent) {
-            console.log(`${i}: ${player_name || player_codes[player_num]} (${player_character}) beat ${opponent_name || opponent_code} (${opponent_character}) in ${game_length}! (${files[i]})`)
-            total_wins++
-            total_games++
-            character_totals[player_character_num] = (character_totals[player_character_num] + 1) || 1
-            character_wins[player_character_num] = (character_wins[player_character_num] + 1) || 1
-            nickname_totals[player_name] = (nickname_totals[player_name] + 1) || 1
-            nickname_wins[player_name] = (nickname_wins[player_name] + 1) || 1
-            opponent_totals[opponent_code] = (opponent_totals[opponent_code] + 1) || 1
-            opponent_wins[opponent_code] = (opponent_wins[opponent_code] + 1) || 1
-            stage_totals[stage_num] = (stage_totals[stage_num] + 1) || 1
-            stage_wins[stage_num] = (stage_wins[stage_num] + 1) || 1
+            console.log(`${i}: ${player_name || player_codes[player_num]} (${player_character}) beat ${opponent_name || opponent_code} (${opponent_character}) in ${game_length}! (${file})`)
+            data.total_wins = 1
         } else {
-            console.log(`${i}: ${player_name || player_codes[player_num]} (${player_character}) lost to ${opponent_name || opponent_code} (${opponent_character}) in ${game_length}. (${files[i]})`)
-            total_games++
-            character_totals[player_character_num] = (character_totals[player_character_num] + 1) || 1
-            nickname_totals[player_name] = (nickname_totals[player_name] + 1) || 1
-            opponent_totals[opponent_code] = (opponent_totals[opponent_code] + 1) || 1
-            stage_totals[stage_num] = (stage_totals[stage_num] + 1) || 1
+            console.log(`${i}: ${player_name || player_codes[player_num]} (${player_character}) lost to ${opponent_name || opponent_code} (${opponent_character}) in ${game_length}. (${file})`)
         }
 
+        data.total_games = 1
+        data.player_character_num = player_character_num
+        data.player_name = player_name
+        data.opponent_code = opponent_code
+        data.stage_num = settings.stageId
+
         // Try to find last used nickname and actual connect code to display at the end
         if (player_name.length > 0) {
-            final_player_name = player_name
+            data.final_player_name = player_name
         }
-        real_player_code = player_codes[player_num]
+        data.real_player_code = player_codes[player_num]
         if (opponent_arg && player_names[opponent_num]) {
```

Comment:
```suggestion
        if (player_names[opponent_num]) {
```
Remove opponent_arg check before saving final_opponent_name or opponents' names will only be saved if doing a head to head search